banner-plugin/package.json

```diff
@@ -52,5 +52,6 @@
   },
   "engines": {
     "node": ">=20"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha1.ac34549e6aa8e7ead463a7407e1c7390f61a6610"
```

Comment:
The packageManager will limit us to use yarn, I think we can keep it because yarn guarantees cross platform support and guarantee that projects work in future as well